.github/ISSUE_TEMPLATE/5_KIT_template.md

```diff
@@ -0,0 +1,59 @@
+---
+name: Propose KIT
+labels: kit
+---
+
+# Overview
+
+## Description
+<!-- Please provide a short description -->
+
+## Impact
+<!-- Please name the impact -->
+- Impact 1
+
+## KIT Team
+
+### Contributor
+<!-- names are already needed for open planning -->
+- Contributor 1
+- Contributor 2
+
+### Committer
+<!-- names are already needed for open planning -->
```

Comment:
Something I'm always unaware of: who is the `assignee` of the ticket? Is it the responsible or the responsible committers + contributors? It might might sense to state that in the comment in here that the committers and / or contributers should also be assigned (increases sanity).